### PR TITLE
Update deps.

### PR DESCRIPTION
**Pull request by matthewbauer**
I'm just going to leave this here to see if any problems come up.

### CODE REVIEW

**Comment by xaiki** [comment]
this needs to be throughly tested, especially chromecast/airplay support maybe @guerrerocarlos can have a look ?

File: package.json [hunk 1]
```diff
@@ -28,64 +28,64 @@
...
-    "nedb": "1.1.2",
-    "node-captions": "0.3.8",
-    "node-tvdb": "1.3.1",
-    "node-webkit-fdialogs": "latest",
-    "nodecast-js": "git+https://github.com/gyzerok/nodecast-js",
+    "adm-zip": "^0.4.7",
+    "airplay-js": "^0.2.16",
+    "async": "^0.9.2",
+    "chromecast-js": "^0.1.7",
+    "gitlab": "^1.4.1",
```
Comment (MrAdder) [inline package.json]: Not Needed

File: package.json [hunk 2]
```diff
@@ -28,64 +28,64 @@
...
-    "semver": "4.x.x",
-    "send": "0.12.2",
-    "strike-api": "0.2.0",
-    "tar": "1.x.x",
-    "temp": "0.x.x",
-    "torrent-tracker-health": "0.x.x",
-    "underscore": "1.x.x",
-    "upnp-mediarenderer-client": "1.x.x",
-    "urijs": "1.x.x",
-    "wcjs-player": "git+https://github.com/butterproject/wcjs-player.git",
```
Comment (MrAdder) [inline package.json]: This is required as is we are building our own

**Comment by MrAdder** [comment]
Dependecies are being updated automatically now

**Pull request closed without merging**